src/ws-dropdown/ws-dropdown.js

```diff
@@ -312,14 +318,26 @@ export class WSDropdown extends Component {
     }
     switch (this.props.type) {
       case 'anchor':
-        return <a className="dropdown-trigger" onClick={() => this.open()}>{icon} {this.state.text}</a>;
+        return <a className={"dropdown-trigger" + this.disabledStyle()}
+          onClick={() => this.open()}>
+            {icon} {this.state.text}
+        </a>;
       case 'button':
-        return <button className="dropdown-trigger" onClick={() => this.open()}>{icon} {this.state.text}</button>;
+        return <button className={"dropdown-trigger" + this.disabledStyle()}
+          onClick={() => this.open()}>
+            {icon} {this.state.text}
+        </button>;
       case 'select':
-        return <div className="dropdown-trigger select-box" onClick={() => this.open()}>{icon} {this.state.text}</div>;
+        return <div className={"dropdown-trigger select-box" + this.disabledStyle()}
+          onClick={() => this.open()}>
+            {icon} {this.state.text}
+        </div>;
       case 'icon':
       default:
-        return <a className="dropdown-trigger" onClick={() => this.open()}>{icon}</a>;
+        return <a className={"dropdown-trigger" + this.disabledStyle()}
+          onClick={() => this.open()}>
```

Comment:
Could you pls execute gulp lint If you didn't already? I'm not sure if the formatting is correct